src/trainer_see.c

```diff
@@ -190,10 +306,29 @@ static const union AnimCmd *const sSpriteAnimTable_Icons[] =
     sSpriteAnim_Icons4
 };
 
+static const union AnimCmd *const sSpriteAnimTable_Emotes[] =
+{
+    sSpriteAnim_Emotes0,
+    sSpriteAnim_Emotes1,
+    sSpriteAnim_Emotes2,
+    sSpriteAnim_Emotes3,
+    sSpriteAnim_Emotes4,
+    sSpriteAnim_Emotes5,
+    sSpriteAnim_Emotes6,
+    sSpriteAnim_Emotes7,
+    sSpriteAnim_Emotes8,
+    sSpriteAnim_Emotes9,
+    sSpriteAnim_Emotes10,
+};
+
+// TODO: Move these declarations into even_object_movement.h
```

Comment:
Todo reminder

Reply:
Y'all probably already did this

Reply:
I don't think we did

Reply:
Don't you have pal tag definitions in some header file, though?

Reply:
oh yeah, `include/constants/event_objects.h`. I was confused when you mentioned a different file, sorry ^^;
I'll move these now.

Reply:
Ah nvm, they were already there, so I'll just remove these duplicates :)